### PR TITLE
Qualifying DC/OS 1.13.9 against CentOS 7.8 and Docker 18.09.9 and docker 19.03.5

### PR DESCRIPTION
**Pull request by DominikDary**

### CODE REVIEW

**Comment by DominikDary** [comment]
@fatz could you please review the pr?

File: rhel/7.8/aws/DCOS-1.13.9/docker-18.09.9-ce/selinux_enforcing/cluster_profile.tfvars [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+ssh_key_name = "dcos-images"
+# Inbound Master Access
+admin_cidr = "0.0.0.0/0"
+
+num_of_masters = "1"
+num_of_private_agents = "2"
+num_of_public_agents = "1"
+
+custom_dcos_download_path = "https://downloads.dcos.io/dcos/testing/1.13.8/commit/900d88f095d19882ac4ff97f56ac0394e30f5392/dcos_generate_config.sh"
```
Comment (gauripowale) [inline rhel/7.8/aws/DCOS-1.13.9/docker-18.09.9-ce/selinux_enforcing/cluster_profile.tfvars:16]: Wrong dcos_generate_config.sh used, should be for 1.13.9 not 1.13.8

File: rhel/7.8/aws/DCOS-1.13.9/docker-19.03.5-ce/selinux_enforcing/cluster_profile.tfvars [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+ssh_key_name = "dcos-images"
+# Inbound Master Access
+admin_cidr = "0.0.0.0/0"
+
+num_of_masters = "1"
+num_of_private_agents = "2"
+num_of_public_agents = "1"
+
+custom_dcos_download_path = "https://downloads.dcos.io/dcos/testing/1.13.8/commit/900d88f095d19882ac4ff97f56ac0394e30f5392/dcos_generate_config.sh"
```
Comment (gauripowale) [inline rhel/7.8/aws/DCOS-1.13.9/docker-19.03.5-ce/selinux_enforcing/cluster_profile.tfvars:16]: Should be using the 1.13.9 dcos_generate_config.sh

File: rhel/7.8/aws/DCOS-1.13.9/docker-18.09.9-ce/selinux_enforcing/cluster_profile.tfvars (hunk 1 above)
Comment (DominikDary) [reply rhel/7.8/aws/DCOS-1.13.9/docker-18.09.9-ce/selinux_enforcing/cluster_profile.tfvars]: Good catch, ty

File: rhel/7.8/aws/DCOS-1.13.9/docker-19.03.5-ce/selinux_enforcing/cluster_profile.tfvars (hunk 2 above)
Comment (DominikDary) [reply rhel/7.8/aws/DCOS-1.13.9/docker-19.03.5-ce/selinux_enforcing/cluster_profile.tfvars]: Good catch, ty

**Pull request merged**